src/gravatar/web_api.js

```diff
@@ -1,8 +1,10 @@
-import { md5 } from 'blueimp-md5';
+import blueimp from 'blueimp-md5';
 import { validateEmail } from '../cred/index';
 import jsonp from '../utils/jsonp_utils';
 import * as preload from '../preload/index';
 
+const md5 = blueimp.md5 || blueimp;
```

Comment:
Question: why you need to load it like that? it there some inconsistency on blueimp? (it is ok to me, just seemed weird)